tests/Decoder.js

```diff
@@ -155,6 +155,18 @@ test('Decode list arguments', t => {
     t.deepEqual(decoded, ints, 'test_list([1, 2, 3, 5, 8, 13, 21])')
 })
 
+test('Decode long list arguments', t => {
+    t.plan(1)
+    const decoded = encoder.decode(CONTRACT, 'test_list', 'cb_HwFUVFRUVFRUVFRUVFRUVFRUVBiJdZ4=')
```

Comment:
@davidyuk how did you endup with cb_HwFUVFRUVFRUVFRUVFRUVFRUVBiJdZ4 ?

Reply:
It can be reproduced at https://github.com/aeternity/aepp-calldata-js/pull/149:

```
./tests/contract-call.sh test_list "[[42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42]]"
test_list([[42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42]]) -> cb_HwFUVFRUVFRUVFRUVFRUVFRUVBiJdZ4=
```